### DISCUSSION TITLE
App Center: Uprawnienia/ AAD/ Płatności

### DISCUSSION THREAD

**Discussion by phaate** [AzureCommunityPL/group#168] [Pomoc] [answered]
Cześć!

Mam paru Devów, którzy zainteresowani są używaniem App Center ale martwi mnie parę rzeczy.

Da się to zintegrować z AAD w ten sposób, że do organizacji będą miały "wjazd" tylko osoby w danej grupie?
Czy jak podepne się subskrypcją i ustale "plan" to inny admin może się bawić suwakiem i zmienić sobię plan na wyższy?

Te uprawnienia wewnątrz aplikacji są dość ubogie. Macie może jakieś linki/ tutoriale do tego?

**Comment by wi5nia** [comment, marked as answer]
Sa ubogie i niewiele da sie zrobic. Generalnie to podpinasz subskrypcje Azure do App Center i masz Adminow, ktorzy moga zmieniac plany w AppCenter, a calej reszcie - czytaj dev dajesz max contributora albo member

**Reply by phaate** [reply]
Tak myslalem bo gdzies tam testowo klikalem. 
Dzięki!
